### PR TITLE
Quick fix for action group_followee_count

### PR DESCRIPTION
**Pull request by wildcatzita**
### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [X] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

**Comment by wildcatzita** [comment]
If my quick fix will be accepted - I'll update the API Documentation.

**Comment by tino097** [comment]
can you please provide  test coverage

**Comment by wildcatzita** [comment]
@tino097 , Sorry forgot to write API call. URL is /api/action/group_followee_count?id=user_id&type=organization - it will filter by organization.

File: ckan/model/follower.py [hunk 1]
```diff
@@ -85,6 +115,7 @@ def _get(cls, follower_id=None, object_id=None):
                 cls.object_id == object_alias.id,
                 follower_alias.state != core.State.DELETED,
                 object_alias.state != core.State.DELETED,
+                object_alias.type == 'organization',
```
Comment (wardi) [inline ckan/model/follower.py]: `object_alias.is_organization == True` is better here

File: ckan/logic/action/get.py [hunk 2]
```diff
@@ -3047,6 +3047,9 @@ def _followee_count(context, data_dict, FollowerClass):
         data_dict, errors = _validate(data_dict, schema, context)
         if errors:
             raise ValidationError(errors)
+    if data_dict.get('__extras'):
```
Comment (amercader) [inline ckan/logic/action/get.py]: We should never rely on `__extras` as these are just byproducts of the validation process. It's better if whatever function that is calling `_followee_count` passes an explicit `is_organization` parameter (that defaults to False)

**Comment by wildcatzita** [comment]
@amercader , @wardi , Fixed.

For filter by organization added to dict _'is_organization': True_ (by default it will be False):
`get_action('group_followee_count')(context, {'id': user_id, 'is_organization': True)`

**Comment by kmbn** [comment]
We decided to close old pull requests that are not actively worked on so that we can focus our effort and attention on PRs made against the current version of CKAN. A lot has changed in the last several months and many PRs will have gone stale or may now be in conflict with the latest version.

Please note: this isn't a judgement about the quality of your pull request. We just don't have the resources to review older PRs.

If you would like to reopen your PR (and update it if necessary), please do!

**Pull request closed without merging**